docs/ref/request-response.txt

```diff
@@ -396,6 +396,23 @@ Methods
     Returns ``True`` if the request is secure; that is, if it was made with
     HTTPS.
 
+.. method:: HttpRequest.accepts(mime_type)
+
+    .. versionadded:: 3.1
+
+    Returns ``True`` if the request ``Accept`` header matches the ``mime_type``
+    argument::
+
+        >>> request.accepts('text/html')
+        True
+
+    If a response varies depending on the content of the ``Accept`` header and
+    you are using some form of caching like Django's :mod:`cache middleware
+    <django.middleware.cache>`, you should decorate the view with
+    :func:`vary_on_headers('Accept')
+    <django.views.decorators.vary.vary_on_headers>` so that the responses are
+    properly cached.
```

Comment:
Maybe a quick extra paragraph:

```
Most browsers send ``Accepts: */*`` by default, so this would return ``True`` for all content types. Setting an explicit ``Accepts`` in API requests can be useful for returning a different content type for those consumers only. See `Content negotiation example <content-negotiation-example>`_ for an example of using ``accepts()`` to return different content to API consumers.
```